discordbot/commands/drc.py

```diff
@@ -82,29 +96,32 @@ async def matchups(ctx: MtgContext, *, args: str) -> None:
         domain=link_domain, length=output['length'], wr=output['winrate'], url=output['compress'])
     await ctx.send(ans)
 
-def search_dreadrise(query: str) -> Tuple[int, List[str]]:
+async def search_dreadrise(query: str) -> Union[Tuple[str, None], Tuple[int, List[str]]]:
     """Returns a tuple. First member is an integer indicating how many cards match the query total,
        second member is a list of card names up to the maximum that could be fetched in a timely fashion."""
     if not query:
         return 0, []
 
     query = fetch_tools.escape(query)
     url1 = f'{domain}/cards/find?q={query}+output:resultcount'
+    count_txt = await fetch_tools.fetch_async(url1)
     try:
-        count = int(fetch_tools.fetch(url1))
+        count = int(count_txt)
     except ValueError:
-        return 0, []
+        return count_txt, None  # this sucks but i need 2 values to unpack
```

Comment:
This would then be `return 0, [], count_txt` or `return 0, [], f'Failed to get count from {count_txt}'`